redwood-treehouse-host/src/commonMain/kotlin/app/cash/redwood/treehouse/TreehouseAppContent.kt

```diff
@@ -291,6 +291,9 @@ private class ViewContentCodeBinding<A : AppService>(
     if (canceled) return
 
     viewOrNull = view
+
+    view.saveCallback = this
```

Comment:
I wanna carefully lifecycle manage this. We should be able to guarantee that there’s always exactly one `TreehouseAppContent` that drives each `TreehouseView`, AND that this assignment does the right thing when code is hot-reloaded.

Unfortunately I’ve done a pretty abysmal job making any of this code testable, so testing it is going to be awkward.

That’s a thing I gotta fix. Happy to have help too.

Reply:
added to a list of TODOs (to be posted as a comment to this PR)